### ISSUE TITLE
pop-over: repository name illegible in dark mode

### ISSUE THREAD

**Issue by shinenelson** [tanmayrajani/notifications-preview-github#119]
the name of the repository is illegible in dark mode
![Screenshot from 2021-01-17 09-30-01](https://user-images.githubusercontent.com/4771718/104830849-a02d3780-58a8-11eb-9eb0-22d77898cdca.png)

My guess is that because the following code unsets `color` for `data-color-mode=dark`
https://github.com/tanmayrajani/notifications-preview-github/blob/8e07a8c2ad142b346406b3c2d7d49d4abaa9669a/extension/github-notifications-preview.css#L10-L12

GitHub's `.text-inherit` probably cannot inherit a `color` since it was unset earlier. Removing the class from the anchor node falls back to the default blue anchor color from GitHub's `base.scss` ( which is also okay in my humble opinion ).

PS : I can create a pull request if pointed in the right direction

**Comment by tanmayrajani** [comment]
could you please try to update your extension and see if that solves the problem? I see it correctly so
![image](https://user-images.githubusercontent.com/8295888/104832379-acb88c80-58b6-11eb-8fbe-ba9dc02ea284.png)

**Comment by shinenelson** [comment]
I believe I am on the [latest release](https://github.com/tanmayrajani/notifications-preview-github/releases/latest) (which introduced dark mode support)

![image](https://user-images.githubusercontent.com/4771718/104832844-45044080-58ba-11eb-851c-0be86dca5c85.png)

To avoid the possibility of cache, I signed in from Private Browsing and found the issue to still exist.
To avoid the possibility of other extensions clashing with this one, I also created a fresh profile and installed this extension as the only extension in the profile. The issue still existed.

Also, the action buttons do not look as in your screenshot for me either. It looks like the screenshot in #116 (as if the extension was not updated at all).
![Screenshot from 2021-01-17 12-13-02](https://user-images.githubusercontent.com/4771718/104833320-b85b8180-58bd-11eb-8f53-06334dca3d13.png)

For what it is worth, this is on Firefox. I extracted the extension package (`xpi`) and found the CSS files are similar in terms of content ( though not identical due to indentation differences ). I could not compare the javascript file because it was transpiled from JSX.
